release-notes/opensearch-dashboards.release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,98 @@
+* [CVE] Upgrades babel, storybook, and postcss ([#1104](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1104))
+* [CVE] Upgrade immer from 8.0.1 to 9.0.6 ([#780](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/780))
+
+### Features/Enhancements
+* Adds src/test_utils/ and src/core paths to the target clean patterns ([#1442](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1442))
+* Adds code coverage and GitHub Actions badges ([#1498](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1498))
+* Adds code coverage reporting ([#1478](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1478))
+* [Feature] Expressions example plugin ([#1438](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1438))
+* [Circuit-Breaker] Add memory circuit breaker configuration ([#1347](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1347))
+* Fixes search usage telemetry ([#1427](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1427))
+* [Branding] allow for SSL setup failures ([#1414](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1414))
+* [Improvement] node healthcheck optimization ([#1386](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1386))
+
+### Bug Fixes
+* Fix: Re-rendering visualization when expression changes and improves typing ([#1491](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1491))
+* fix(actions): Better type checks for icons ([#1496](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1496))
+* [Bug] fix copy as curl ([#1472](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1472))
+* update re2 build for arm under node 14 ([#1454](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1454))
+* update re2 for linux, darwin, and windows ([#1453](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1453))
+* fix(Tooltip): Fixes tooltip when split series charts are used ([#1324](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1324))
+* bumps chromedriver to v100 ([#1410](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1410))
+* Fixes the header's nav trigger button not closing the nav ([#1394](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1394))
+* Fixes the linting rules to accept only the approved copyright headers ([#1373](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1373))
+
+### Infrastructure
+* [Plugins] fix default path to plugins ([#1468](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1468))
+* Reverts re2 back to 1.15.4 from 1.17.4 to fix build issues ([1419](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1419))
+* [CI][Build] Use BUILD_NUMBER for building bundles ([#1371](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1371))
```

Comment:
can be removed for since it came in 1.3.1

Reply:
will remove